src/pages/SignUp/SignUp.jsx

```diff
@@ -1,7 +1,259 @@
+                  id="email"
+                  type="text"
+                  className="input-box"
+                  onChange={handleEmailInput}
+                ></input>
+                {email.length === 0 || isEmailValid || (
+                  <div className="input-demand signup-right-font">
+                    이메일 형식이 아닙니다.
+                  </div>
+                )}
+                <div className="space"></div>
+                <span className="name-title">비밀번호</span>
+                <input
+                  id="password"
+                  type="password"
+                  className="input-box"
+                  onChange={handlePwInput}
+                ></input>
+                {password.length === 0 || isPasswordValid || (
+                  <div className="input-demand signup-right-font">
+                    비밀번호는 8자 이상, 영문자,숫자,특수문자 포함입니다.
+                  </div>
+                )}
+                <div className="space"></div>
+                <span className="name-title">휴대폰</span>
+                <input
+                  id="phone"
+                  type="text"
+                  className="input-box"
+                  onChange={handlePhoneNumberInput}
+                ></input>
+                {phoneNumber.length === 0 || isPhoneNumberValid ? (
+                  <div className="input-demand-phone signup-right-font">
+                    010-1234-5678 or 01012345678
+                  </div>
+                ) : (
+                  <div className="input-demand signup-right-font">
+                    휴대폰 번호가 올바르지 않습니다.
+                  </div>
+                )}
+                <div className="space"></div>
+                <button class="address-button">우편번호 찾기</button>
+                <div className="space"></div>
+                <span className="name-title">주소</span>
+                <input
+                  id="address"
+                  type="text"
+                  className="input-box"
+                  onChange={handleAddressInput}
+                ></input>
+                <div className="space"></div>
+                <div class="checkbox">
+                  <input type="checkbox" className="input-checkbox"></input>
+                  <div className="input-description">
+                    마케팅 수신 동의를 통해 IKEA의 홈퍼니싱 아이디어와 신상품
+                    소식, 그리고 할인 혜택 정보를 받고 싶어요!
+                  </div>
+                </div>
+                <div class="checkbox">
+                  <input type="checkbox" className="input-checkbox"></input>
+                  <div>
+                    (필수) 약관을 모두 읽고 동의합니다.
+                    <a target="_blank" href="#" class="link">
+                      이용약관
+                    </a>
+                    .
+                  </div>
+                </div>
+                <div class="checkbox">
+                  <input type="checkbox" className="input-checkbox"></input>
+                  <div>
+                    (필수) 개인정보 수집ㆍ이용에 동의합니다.{' '}
```

Comment:
```suggestion
                    (필수) 개인정보 수집ㆍ이용에 동의합니다.
```

불필요한 코드가 들어가있진 않은지 항상 확인하기!